eos/constraints/ee-to-ccbar.yaml

```diff
@@ -0,0 +1,2783 @@
+        - [  0,0,0,-0.000022,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.00026,0,0,0,0,0 ]
+        - [ -0.000053,0.000033,0.00053,0.0018,0.00055,0,0.000085,0.000094,0.00010,0.00012,0.00015,0.00012,0.000079,0.000048,0.000031,0.000043,0.000089,0.00010,0.00016,0.00015,0.00010,0.00018,0.000081,0.000023,0.000047,0.000049,0.000046,0.000019,0.000027,0.000020,0.000010,0.000031,0.000016,0.000033,0.000019,0.000053,0.000071,0.000038,0.000018,0.000049,0.000023,0.000048,0.000053,0.000018,0,0.000020,0,0.000013,0.000019,0.000018,0.000016,0.000026,0.000020,0.000013,0.000018,0.000020,0,0.000013,0.000023,0,0.00074,0,0.000024,0,0.000029 ]
+        - [  0,0,0.000064,0.00022,0.000066,0,0.000010,0.000011,0.000013,0.000015,0.000018,0.000014,0,0,0,0,0.000010,0.000012,0.000020,0.000018,0.000012,0.000022,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.00028,0,0,0 ]
+        - [ -0.000045,0.000029,0.00045,0.0015,0.00047,0,0.000072,0.000080,0.000093,0.00011,0.00013,0.00010,0.000068,0.000041,0.000027,0.000036,0.000076,0.000090,0.00014,0.00013,0.000088,0.00016,0.000070,0.000020,0.000040,0.000042,0.000040,0.000016,0.000023,0.000017,0,0.000026,0.000014,0.000028,0.000016,0.000045,0.000061,0.000032,0.000015,0.000042,0.000020,0.000041,0.000045,0.000015,0,0.000017,0,0.000011,0.000016,0.000015,0.000013,0.000022,0.000017,0.000011,0.000015,0.000017,0,0.000011,0.000019,0,0.000024,0,0.00072,0,0.000025 ]
+        - [  0,0,-0.000070,-0.00024,-0.000072,0,-0.000011,-0.000012,-0.000014,-0.000017,-0.000020,-0.000016,-0.000010,0,0,0,-0.000011,-0.000014,-0.000022,-0.000020,-0.000013,-0.000025,-0.000010,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.00014,0 ]
+        - [ -0.000056,0.000035,0.00055,0.0019,0.00057,0,0.000089,0.000098,0.00011,0.00013,0.00016,0.00012,0.000083,0.000051,0.000033,0.000045,0.000093,0.00011,0.00017,0.00016,0.00010,0.00019,0.000085,0.000024,0.000049,0.000052,0.000049,0.000020,0.000028,0.000021,0.000011,0.000032,0.000017,0.000034,0.000019,0.000056,0.000074,0.000040,0.000018,0.000051,0.000024,0.000051,0.000056,0.000018,0,0.000021,0,0.000014,0.000020,0.000019,0.000017,0.000027,0.000021,0.000014,0.000019,0.000021,0,0.000014,0.000024,0,0.000029,0,0.000025,0,0.00086 ]
+    dof: 65
+# }}}
+#
+# [BaBar:2007]
+#
+# {{{
+ee->ccbar::sigma_eetoD0Dbar0@BaBar:2007B:
+    type: MultivariateGaussian(Covariance)
+    dim: 89
+    observables:
+        [ ee->ccbar::sigma_eetoD0Dbar0(E), ee->ccbar::sigma_eetoD0Dbar0(E), ee->ccbar::sigma_eetoD0Dbar0(E), ee->ccbar::sigma_eetoD0Dbar0(E),
```

Comment:
Hm, I did not notice this before. Can we change the structure of the observable names to something more consistent with the rest of EOS?

I would suggest ``e^+e^-->D0Dbar0::sigma(E)``, and similar. Sorry for not mentioning this earlier.